database/migrations/2023_12_19_164230_create_user_sessions_table.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::create('user_sessions', function (Blueprint $table) {
+            $table->increments('id');
+            $table->unsignedInteger('user_id');
+            $table->string('user_agent');
+            $table->ipAddress('ip_address');
+            $table->string('device_name', 30);
+            $table->string('device_type', 30);
+            $table->string('device_platform', 30);
+            $table->string('device_browser', 30);
+            $table->string('token');
+            $table->boolean('is_active')->default(false);
+            $table->timestamp('expired_date')->nullable()->default(null);
+            $table->timestamps();
+
+            $table->foreign('user_id')->references('id')->on('users')->onDelete('cascade');
+            $table->index(['token']);
```

Comment:
maybe we can add an index to 'is_active' field, because many queries are filtering by this field and the sessions table will have big data

Reply:
The `is_active` field was indexed